unimport/refactor.py

```diff
@@ -129,9 +129,11 @@ def get_star_imp():
                     ):
                         return imp
 
-            return self.leave_StarImport(
-                original_node, updated_node, get_star_imp()
-            )
+            imp = get_star_imp()
+            if imp:
+                return self.leave_StarImport(original_node, updated_node, imp)
+            else:
+                return original_node
```

Comment:
```suggestion
            return original_node
```
I think the readability won't be dropped when we remove this line!

Reply:
The Python community prefers this structure because the community finds this style easier to read, so it's better this style.